tmux-gitbar.tmux

```diff
@@ -3,15 +3,22 @@
 # Created by Aurélien Rainone
 # github.com/aurelien-rainone/tmux-gitbar
 
-# generate tmux-gitbar.conf if it doesn't exist
-if-shell 'test -z ${TMUX_GITBAR_DIR}' \
-  "if-shell '! stat ~/.tmux-gitbar/tmux-gitbar.conf' \
-    'run-shell \"~/.tmux-gitbar/lib/generate-config.sh ~/.tmux-gitbar/tmux-gitbar.conf\"'" \
-  "if-shell '! stat $TMUX_GITBAR_DIR/tmux-gitbar.conf' \
-    'run-shell \"$TMUX_GITBAR_DIR/lib/generate-config.sh $TMUX_GITBAR_DIR/tmux-gitbar.conf\"'"
+# Set defaults
+# FIXME: Why can I only get this to work if I double nest?
+if-shell 'test True' \
+  "if-shell 'test -z \"${TMUX_GITBAR_DIR}\"' \
+    'TMUX_GITBAR_CONF=\"$HOME/.tmux-gitbar.conf\"' \
+    'TMUX_GITBAR_CONF=\"$TMUX_GITBAR_CONF\"'"
```

Comment:
Wait, you mean the tmux method works on your machine? Even after having killed the tmux server in order to clear the tmux env?